project_material/static/src/fix_editable_list.js

```diff
@@ -16,7 +16,7 @@
     However, Odoo does not prioritize bugs which can not be reproduced in a vanilla
     environment.
 */
-odoo.define("project_wip_material.fix_editable_list", function(require) {
+odoo.define("project_material.fix_editable_list", function(require) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['odoo' is not defined.](https://app.codacy.com/manual/numigi/odoo-project-addons/pullRequest?prid=4797804)